android/build.gradle

```diff
@@ -25,10 +25,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    // Conditional for compatibility with AGP <4.2.
-    if (project.android.hasProperty("namespace")) {
-        namespace 'dev.google.webcrypto'
-    }
+    namespace 'dev.google.webcrypto'
```

Comment:
Dumb question, but do we not need this anymore?

Will `minSdkVersion 21` make AGP < 4.2 irrelevant?

Reply:
AGP 8.4 rejects `AndroidManifest.xml` files with a `package` attribute. If we only set the namespace for AGP ≥ 4.2, then on AGP < 4.2 we won't define any namespace at all and the build will fail anyways.

I think it's reasonable to ask library users to upgrade AGP ≥ 4.2 when upgrading webcrypto.dart.